GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -48,16 +59,38 @@ public static ArgumentString SubmoduleSyncCmd(string? name)
             };
         }
 
-        public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath, string branch, bool force)
+        public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath, string branch, bool force, IEnumerable<GitConfigItem> configs = null)
         {
-            return new GitArgumentBuilder("submodule")
+            ArgumentBuilder cfgArgs = new();
```

Comment:
unused

Reply:
![image](https://user-images.githubusercontent.com/3629489/201679490-6ed95257-607d-449a-b59b-9ac53ab72398.png)
![image](https://user-images.githubusercontent.com/3629489/201680001-befc768b-895f-4d32-b2b5-a94c0a0184d6.png)

?

I am planning on the second pull request to call it more.  Please pull the code down and use CodeLens.

Reply:
This pull request is just for making tests green.  Another will be created to handle actual application functionality.